### DISCUSSION TITLE
Welcome to LANraragi_Chinese Discussions!

### DISCUSSION THREAD

**Discussion by WindyCloudCute** [WindyCloudCute/LANraragi_Chinese#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aicynide** [comment]
Github is banned in china how did you access it?

**Reply by WindyCloudCute** [reply]
It's not completely banned, it's just that it can't be accessed normally in some areas.
